components/domobject_derive/lib.rs

```diff
@@ -2,27 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#![recursion_limit = "128"]
```

Comment:
Why is this now needed?

Reply:
Because quote! recurses heavily and it failed to compile without this

Reply:
I'll try removing it to see if it was during a temporary state

Reply:
Doesn't compile without it, if that's a problem I guess I could split the quote! invocation in 2...

```
error: recursion limit reached while expanding the macro `stringify`
  --> components/domobject_derive/lib.rs:36:21
   |
36 |       let mut items = quote! {
   |  _____________________^
37 | |         impl #impl_generics ::js::conversions::ToJSValConvertible for #name #ty_generics #where_clause {
38 | |             #[allow(unsafe_code)]
39 | |             unsafe fn to_jsval(&self,
...  |
58 | |         }
59 | |     };
   | |_____^
```